Docs/Generali/piano_di_qualifica/01_intro.tex

```diff
@@ -0,0 +1,27 @@
+\section{Introduzione}
+    \subsection{Scopo del documento}
+        Questo documento ha lo scopo di descrivere nel dettaglio i principi guida stabiliti e le operazioni effettuate
+        dal team \textit{Six Bit Busters} per garantire la qualità dei processi e dei prodotti nello sviluppo del progetto
+        \textit{3Dataviz}. In particolare, esso descrive i risulati delle analisi ottenute tramite le varie misurazioni
+        della qualità dell'operato del team facendo quindi emergere eventuali problematiche riscontrate e di conseguenza
+        applicare delle correzioni opportune.
+
```

Comment:
Questo documento descrive in modo approfondito i principi guida e le attività messe in atto dal team SixBitBusters per assicurare l'eccellenza dei processi e dei prodotti sviluppati nel progetto 3Dataviz. In particolare, vengono presentati i risultati delle analisi qualitative condotte per valutare le performance del team, evidenziando eventuali criticità e descrivendo le azioni correttive intraprese.